package.json

```diff
@@ -31,77 +31,77 @@
   },
   "dependencies": {
     "@paralect/common-logger": "0.1.10",
-    "@paralect/node-mongo": "0.3.1",
```

Comment:
I like this, it is much more clear to talk to an API directly instead of the database.